### ISSUE TITLE
Development build cannot be run

### ISSUE THREAD

**Issue by CrafterSvK** [liberapay/liberapay.com#1961]
When I try to use `make env` in root directory of repository this comes up:
```
/bin/sh: riadok 1: : príkaz nenájdený
"" -m venv env
/bin/sh: riadok 1: : príkaz nenájdený
make: *** [Makefile:18: env] Error 127
```

**Comment by Changaco** [comment]
It looks like you don't have Python 3.6 or 3.7 installed. I'm going to improve the Makefile to at least provide a proper error message.

**Comment by Changaco** [comment]
You can try `make env python=python`, but I can't guarantee that using a Python version other than 3.6 will work.

**Comment by CrafterSvK** [comment]
I do have Python 3.9 on ArchLinux, there is no python3 binary, that must be a source of the problem

**Comment by Changaco** [comment]
No. Arch does have a `/usr/bin/python3` file, but Liberapay's `Makefile` currently looks for `python3.6` and `python3.7` only, not `python3`.

**Comment by CrafterSvK** [comment]
Oh indeed, I didn't know.

**Comment by Changaco** [comment]
Fixed in #1959.
